resources/help.txt

```diff
@@ -1154,7 +1154,7 @@ Starts a remote server to let you run your app in the iOS Simulator from a Windo
 
 This operation is applicable only to OS X systems.
 
-IMPORTANT: If you want to listen on a port in the range of 0 through 1023, run the command using sudo. For example: $ sudo appbuilder remote 443 
+IMPORTANT: You cannot listen on a port in the range of 0 through 1023. 
```

Comment:
Drop this line. Instead, fix the info for \<Port\>

\<Port\> is an integer greater than X that specifies a port on your OS X system.